src/ipn/cart.ts

```diff
@@ -0,0 +1,41 @@
+import { IPN_TYPES_ENUM } from '../constants'
+import {
+  BaseIPNFields,
+  PaymentStatusFields,
+  BuyerInformationFields,
+  ShippingInformationFields,
+  CoinpaymentsIPN
+} from '../types'
+
+// Cart
+export type CartFields = {
+  ipn_type: typeof IPN_TYPES_ENUM.CART
+  txn_id: string
+  currency1: string
+  currency2: string
+  amount1: string
+  amount2: string
+  subtotal: string
+  shipping: string
+  tax: string
+  fee: string
+  [key: `item_name_${number}`]: string
```

Comment:
this is pretty smart syntax, never saw it before

Reply:
Magic